src/CritCore/Translations/Translations.js

```diff
@@ -92,9 +92,11 @@ let translations = {
       userName: 'Nombre de usuario',
       password: 'Ingrese contraseña',
       repeatPassword: 'Repita contraseña',
-      alterEgo: 'Nombre visible / alter ego',
+      alterEgo: 'Nombre visible / Alter ego',
       icon: 'Elija un avatar',
-      submit: 'Crear'
+      submit: 'Crear',
+      passwordTitle : '7-30 letras & numeros',
```

Comment:
Please fix this orthographic errors.

Reply:
Done.

Reply:
This isnt fixed. You must change it to '7-30 letras & números'. The same for the next line.